web/themes/joinup/templates/field/field--fieldset--show-more.html.twig

```diff
@@ -0,0 +1,44 @@
+{#
+/**
+ * @file
+ * Override.
```

Comment:
A restricted description from someone that pays quite a bit of attention. Update this please :)

Reply:
Ouch :)